File: tests/fixtures/sample.ts
// tests/fixtures/sample.ts
export function greet(name: string): string {
  return `Hello, ${name}`;
}

export class AuthService {
  login(userId: number, password: string): boolean {
    return password === "secret";
  }

  logout(userId: number): void {}
}

const helper = () => "helper";

File: tests/fixtures/sample.py
# tests/fixtures/sample.py
def greet(name: str) -> str:
    return f"Hello, {name}"


class AuthService:
    def login(self, user_id: int, password: str) -> bool:
        return password == "secret"

    def logout(self, user_id: int) -> None:
        pass


def _private_helper() -> None:
    pass
